crates/tabby-scheduler/queries/ruby.scm

```diff
@@ -0,0 +1,66 @@
+; https://github.com/tree-sitter/tree-sitter-ruby/blob/master/queries/tags.scm
```

Comment:
If we're only using the default tags, you can refer to the exported variable `TAG_QUERY` from tree-sitter-ruby directly, just like the Rust or Python configurations.

Reply:
updated 👍